stripe.go

```diff
@@ -140,6 +144,13 @@ type BackendConfig struct {
 	URL string
 }
 
+// RequestMetrics contains the payload sent in the `X-Stripe-Client-Telemetry`
+// header when stripe.EnableTelemetry = true.
+type RequestMetrics struct {
```

Comment:
Let's call this `requestMetrics` instead so that it's not exported outside the package (whether the first letter is lower case or upper case determines this).

Reply:
I initially did make `RequestMetrics` private, however I wanted the tests to ensure that I could unmarshal the sent metrics back into this struct. Do you think it's ok to omit that check? Otherwise, I could duplicate the `requestMetrics` definition in `stripe_test.go` for the sake of unmarshaling.

Reply:
Oops, sorry I missed this:

I actually forgot that `stripe_test.go` is in its own `stripe_test` package. IMO, we should probably just put it in `stripe`. All the tests in subpackages are just in the same namespace as their accompanying file (`customer/`, `charge/`, `form/`, etc.) and even most of the `*_test.go` in the top-level package are as well.